dashboard/lib/logic/task_grid_filter.dart

```diff
@@ -35,7 +35,9 @@ class TaskGridFilter extends FilterPropertySource {
   void applyMap(Map<String, String> valueMap) {
     if (valueMap != null) {
       for (final MapEntry<String, String> mapEntry in valueMap.entries) {
-        _allProperties[mapEntry.key].stringValue = mapEntry.value;
+        if (_allProperties.containsKey(mapEntry.key)) {
```

Comment:
is this condition needed, what are cases where unexpected keys exist?

Reply:
This is searching all the query parameters. Since this PR adds repo and branch query params, this needed to be updated to no-op on unknown keys.